test_scripts/API/ButtonSubscription/002_Subscribe_CUSTOM_BUTTON_Success.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile App starts registration.
+-- SDL does:
+-- 1) Send Buttons.SubscribeButton for CUSTOM_BUTTON.
+-- 1) Respond SubscribeButton(SUCCESS) to mobile app.
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variable ]]
+local buttonName = "CUSTOM_BUTTON"
+
+--[[ Local function ]]
+local function registerApp(pAppId, pRpc, pButtonName)
+    common.registerAppWOPTU()
+    EXPECT_HMICALL("Buttons." .. pRpc,{ appID = common.getHMIAppId(pAppId), buttonName = pButtonName })
+    :Do(function(_, data)
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+    end)
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+
+runner.Title("Test")
+runner.Step("App registration", registerApp, { 1, "SubscribeButton", buttonName })
+runner.Step("On Button Press " .. buttonName, common.buttonPress, { 1, buttonName })
```

Comment:
CUSTOM_BUTTON must have button id, can be set by sending RPC with custom button, so button press must be processed then some RPC with custom button is processed.
Also buttonPress function must be expanded with customButtonID and 3rd parameter and use it in ButtonPress and Event.